00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+
+export const RoomListComponent = (props) =>
+  <List>
+    {props.rooms.map((room) =>
+      <ListItem button key={room.id}  onClick={(e) => props.onFieldChange('selectedRoom')(room)}>
```

Comment:
Should I place this event handling not inline?

Reply:
OMHO there is a lot of people talking about performance issues around this, but I got not idea if is bull sheet or really true. More than that do you really need curry on the function?

Reply:
Ok, never mind my prior comment.